### ISSUE TITLE
[en] felt->feel

### ISSUE THREAD

**Issue by Tex2002ans** [languagetool-org/languagetool#4387]
Currently, LT doesn't detect this one:

felt->feel
> That decision made her **felt** proud.

That must be one very happy piece of fabric! :)

**Comment by dpelle** [comment]
@Tex2002ans wrote:

> That must be one very happy piece of fabric! :)

I did not understand what this comment meant.

**Comment by udomai** [comment]
> > That must be one very happy piece of fabric! :)
> 
> I did not understand what this comment meant.

"felt" est un type de tissu.

"The decision made her felt proud" signifie "La décision rendit son feutre fier" (au lieu de "The decision made her feel proud" → "La décision la rendit fière")

**Comment by Tex2002ans** [comment]
> @Tex2002ans wrote:
> 
> > That must be one very happy piece of fabric! :)
> 
> I did not understand what this comment meant.

Ahhh. In this case, LT thought "felt" (noun) = a type of fabric:

Example 1:

> That decision made her **feel** proud.

The woman feels proud!

Example 2:

> That decision made her **felt** proud.

The decision made the woman's (fabric) proud!

I doubt there's ever a time anyone would actually mean that number 2! :P

- - -

https://www.merriam-webster.com/dictionary/felt
https://www.lexico.com/en/definition/felt
https://www.collinsdictionary.com/us/dictionary/english/felt

M-W's Definition 1:

> a cloth made of wool and fur often mixed with natural or synthetic fibers through the action of heat, moisture, chemicals, and pressure

But the correct recommendation would be "a typo" for Definition 3:

> past tense and past participle of FEEL

**Comment by MikeUnwalla** [comment]
> The decision made the woman's (fabric) proud!
> I doubt there's ever a time anyone would actually mean number 2!

But this is correct:
> That defect made her felt proud.

The original problem is one of semantics, not grammar.

**Comment by Tex2002ans** [comment]
In researching this a bit further, it seems like:

"made him felt" -> "made him feel"

is a common ESL error:

- https://www.quora.com/Is-it-correct-to-say-He-got-a-fever-which-made-him-felt-chilly-dizzy-and-dry-mouth-that-further-caused-him-felt-like-a-walking-dead-in-English
- https://www.englishforums.com/English/MadeHimFeelVsMadeHimFelt/blkblb/post.htm
- https://www.englishforums.com/English/FeelOrFeltAndWhy/bjzjlk/post.htm
- http://www.eslinstructor.net/sanjosewrites/verb_form/verb_form16.html
  - This one looks to have lots of other great "common error" examples too.
 
- - -

Maybe a rule handling "made + him/her/them + felt":

> It **made her felt** good, so she didn't cry when I flew to the United States.
> She **made him felt** happy.
> She **made them felt** happy.
> He got a fever, which **made him felt** chilly, dizzy, and have dry mouth.

**Comment by udomai** [comment]
>"made him felt" -> "made him feel"

>is a common ESL error

Absolutely! In the same vein as "He did not believed me", for which we have a rule (AUXILIARY_DO_WITH_INCORRECT_VERB_FORM). The solution might be to simply extend that rule to include "made + VBD" (with the necessary conditions and precautions).

**Comment by Tex2002ans** [comment]
Here's the "felt" example from the ESLIntructor.net link:

felt->feel
> As I was leaving Vietnam, I told my daughter I loved her more than I loved her brother. It made her **felt** good, so she didn't cry when I flew to the United States.

And here's the explanation given:

> Causative verbs **make, let, have, help** are followed by the BASE form of the verb.

**Comment by Tex2002ans** [comment]
And here's a few other correct examples: 

https://dictionary.cambridge.org/grammar/british-grammar/fell-or-felt

*Fell* is the past simple of the verb *fall*:

> The snow **fell** all day in big white flakes.
> He slipped and **fell**, hurting his leg quite badly.
> She **fell** in love with him as soon as they met.
> Not: **_She felt in love …_**

*Felt* is the past simple of the verb *feel*:

> I **felt** really sad that they had lost all their money.
> They **felt** his forehead and it was very, very hot.

- - -

https://speakspeak.com/confusing-words/fell-felt

*Fell* is the past simple form of *fall*.
fall ⇒ fell ⇒ fallen

Here are some examples:

> All the fruit **fell from the tree** during the strong winds.
> He slipped from the roof and **fell** three metres **to the ground**.
> Paul and Jemma **fell in love** at first sight.
> Michael **fell off** his bicycle again yesterday.

*Felt* is the past simple form of *feel*.
feel ⇒ felt ⇒ felt

> I **felt better** after I took the medicine.
> The players **felt disappointed** because they lost the game so easily.

- - -

"fall/fell in love" is also a common phrase:

https://www.merriam-webster.com/dictionary/fall%20in%20love

I'm also betting "fell down" + "fell to the ground" are also common:

https://english.stackexchange.com/questions/78210/difference-between-fell-to-the-ground-vs-fell-on-the-ground
